components/CompareCostsTab.client.vue

```diff
@@ -22,6 +22,7 @@
     >
       <CostsTable
         :scenarios="appStore.currentComparison.scenarios"
+        :diffing="diffCostsChart"
```

Comment:
I guess it might be time to rename this `diffCosts` if it now applies to the table too.